demos/common/python/openvino/model_zoo/model_api/README.md

```diff
@@ -0,0 +1,150 @@
+
+2. Install the package in the clean environment with `--force-reinstall` key.
+```sh
+pip install openmodelzoo_modelapi-0.0.0-py3-none-any.whl --force-reinstall
+```
+
+To verify the package is installed, you might use the following command:
+```sh
+python -c "from openvino.model_zoo import model_api"
+```
+
+## Model API Wrappers
+
+The Python* Model API package suggests ready-to-use model wrappers, which implement standardized preprocessing/postprocessing functions per "task type" and might be reused in applications as "black-box" models.
+Also, the simple wrapper interface allows the creation of custom wrappers covering different architectures.
+
+The following tasks can be solved with wrappers usage:
+
+| Task type                  | Model API wrappers |
+|----------------------------|--------------------|
+| Background Matting         | <ul><li>`VideoBackgroundMatting`</li><li>`ImageMattingWithBackground`</li></ul> |
+| Classification             | <ul><li>`Classification`</li></ul> |
+| Deblurring                 | <ul><li>`Deblurring`</li></ul> |
+| Human Pose Estimation      | <ul><li>`HpeAssociativeEmbedding`</li><li>`OpenPose`</li></ul> |
+| Instance Segmentation      | <ul><li>`MaskRCNNModel`</li><li>`YolactModel`</li></ul> |
+| Monocular Depth Estimation | <ul><li> `MonoDepthModel`</li></ul> |
+| Named Entity Recognition   | <ul><li>`BertNamedEntityRecognition`</li></ul> |
+|  Object Detection          | <ul><li>`CenterNet`</li><li>`DETR`</li><li>`CTPN`</li><li>`FaceBoxes`</li><li>`RetinaFace`</li><li>`RetinaFacePyTorch`</li><li>`SSD`</li><li>`UltraLightweightFaceDetection`</li><li>`YOLO`</li><li>`YoloV3ONNX`</li><li>`YoloV4`</li><li>`YOLOF`</li><li>`YOLOX`</li></ul> |
+| Question Answering         |  <ul><li>`BertQuestionAnswering`</li></ul> |
+| Salient Object Detection   |  <ul><li>`SalientObjectDetectionModel`</li></ul> |
+| Semantic Segmentation      |  <ul><li>`SegmentationModel`</li></ul> |
+
+## Model API Adapters
+
+Model API wrappers are executor-agnostic, meaning it does not implement the specific model inference or model loading, instead it can be used with different executors having the implementation of common interface methods in adapter class respectively.
+
+Currently, `OpenvinoAdapter` and `OVMSAdapter` are supported.
+
+#### OpenVINO Adapter
+
+`OpenvinoAdapter` hides the OpenVINO™ toolkit API, which allows Model API wrappers launching with models represented in Intermediate Representation (IR) format.
+It accepts a path to either `xml` model file or `onnx` model file.
+
+#### OpenVINO Model Server Adapter
+
+`OVMSAdapter` hides the OpenVINO Model Server python client API, which allows Model API wrappers launching with models served by OVMS.
+
+Refer to __[`OVMSAdapter`](adapters/ovms_adapter.md)__ to learn about running demos with OVMS.
+
+For OpenVINO Model Server Adapter employment, you need to install the requirements:
+```sh
+pip install <omz_dir>/demos/common/python/requirements_ovms.txt
```

Comment:
can it be done with extra modules in model api install?

Reply:
-r missed in command

Reply:
> can it be done with extra modules in model api install?

I'll investigate it